src/test/java/stepDefinitions/SampleSteps.java

```diff
@@ -26,6 +26,28 @@ public void iAmOnTheHomePage() throws Throwable {
         driver.get("https://kristinek.github.io/site");
     }
 
+    @Given("^I am on the shop page$")
+    public void iAmOnTheShopPage() throws Throwable {
+        driver.get("http://automationpractice.com/index.php");
+    }
+
+    @Then("^I should see shop page title$")
+    public void iShouldSeeShopPageTitle() throws Throwable {
+        assertTrue(driver.findElement(By.className("logo")).isDisplayed());
+    }
+
+    @Then("^I should see shop page contact phone$")
+    public void iShouldSeeShopPageContactPhone() throws Throwable {
+        assertEquals("0123-456-789",
```

Comment:
1. PageObjects should be used.
2. Expected phone number should come from scenario.